tools/filamesh/src/main.cpp

```diff
@@ -155,13 +159,16 @@ void processNode(const aiScene* scene, const aiNode* node, std::vector<Part>& me
                 }
 
                 for (size_t j = 0; j < numVertices; j++) {
-                    quatf q = mat3f::packTangentFrame({tangents[j], bitangents[j], normals[j]});
+                    quatf q;
```

Comment:
Isn't q uninitialized with this constructor? You may want to set it to 0 or something when uv0 == nullptr.

Reply:
Good catch, fixed.